app/src/timer-planning/timer-planning.service.ts

```diff
@@ -257,6 +273,7 @@ export class TimerPlanningService {
                     },
                     where: {
                         team_id: {_eq: "${teamId}"},
+                        ${timerOffIdsString}
```

Comment:
add coma at the end of the line

Reply:
there is a comma in the expression itself